driver/k8s/logicalvolume_service.go

```diff
@@ -140,7 +140,28 @@ func (s *LogicalVolumeService) DeleteVolume(ctx context.Context, volumeID string
 		return err
 	}
 
-	return s.Delete(ctx, lv)
+	err = s.Delete(ctx, lv)
+	if err != nil {
```

Comment:
If err indicates `lv` is not found, we should return nil.

Reply:
@ymmt2005 
I doubt that the client will retry when the connection error has happened, but give up if it is reached to max retry count.
Is it needed to distinguish between not found and other errors?

Reply:
Ah, you already say that is necessary, sorry.

Reply:
Um... It seems already modified to return nil when if err indicates `lv` is not found.
Did I misunderstand something?

https://github.com/topolvm/topolvm/pull/256/files#diff-78915383711edcc7723926712f85a2eb106b00aeb79b852c37d8560c16d7a765R160

Reply:
Sorry, I noticed my misunderstanding.
I fixed it.